webpack.config.prod.js

```diff
@@ -19,7 +19,6 @@ module.exports = Object.assign(prodConfig, {
     path: path.resolve('./static/bundles/'),
     filename: "[name]-[chunkhash].js",
     chunkFilename: "[id]-[chunkhash].js",
-    crossOriginLoading: "use-credentials",
```

Comment:
I think we actually need to set this to "anonymous" because the default is `false`, which disables it entirely.